packages/install-ses/install-ses.js

```diff
@@ -18,3 +18,8 @@ lockdown({ errorTaming: 'unsafe' });
 // Even on non-v8, we tame the start compartment's Error constructor so
 // this assignment is not rejected, even if it does nothing.
 Error.stackTraceLimit = Infinity;
+
+harden(TextEncoder.prototype);
+harden(TextEncoder);
```

Comment:
Huh, why are these not already hardened? These two are non-ECMA names, right? So they should be removed from the Compartment globals by SES..

Reply:
Ah, I see, we're changing the definition of the Compartment provided by import-bundle to include these two objects, and since they're endowments, they must be hardenable (thus `.prototype` must be hardened ahead of time).

We might want to update `docs/vat-environment.md` to reflect the availability of these names to vat code. OTOH we might want to prohibit their access (I can't think of any particularly good reason to withhold it, other than uniformity across platforms that might not provide it). (note: there are inbound changes to `vat-environment.md` from @tyg that I need to merge, so please make any changes to that file in a separate PR).

Is `TextEncoder` available in both Node.js and browser worlds? Just want to make sure this doesn't prevent swingset from working in a browser. (we haven't really tested that yet, apart from accidentally as part of the wallet, but I don't want to accidentally add a blocker)

Reply:
`TextEncoder` and `TextDecoder` arrived in browsers before Node.js. They’re the more webby API, which is the reason I’m leaning on them instead of `Buffer` for Compartment Map’s Zip implementation.